client/testutil/bank/query_allbalances_test.go

```diff
@@ -0,0 +1,24 @@
+package bank
+
+import (
+	"context"
+	gnfdclient "github.com/bnb-chain/gnfd-go-sdk/client/grpc"
+	"github.com/bnb-chain/gnfd-go-sdk/keys"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestBankAllBalances(t *testing.T) {
+	km, err := keys.NewPrivateKeyManager("e3ac46e277677f0f103774019d03bd89c7b4b5ecc554b2650bd5d5127992c20c")
+	assert.NoError(t, err)
+	client := gnfdclient.NewGreenfieldClientWithKeyManager("localhost:9090", "greenfield_9000-121", km)
```

Comment:
I will suggest don't have each testing file for each query, one module one testing file.

Reply:
changed